spec/text-editor-spec.coffee

```diff
@@ -4768,3 +4768,28 @@ describe "TextEditor", ->
       subscription.dispose()
       gutter.destroy()
       expect(payloads).toEqual []
+
+  describe '.setGrammar', ->
+    [registration] = []
+    beforeEach ->
+      waitsForPromise ->
+        atom.packages.activatePackage('language-coffee-script')
+
+    afterEach: ->
+      registration?.dispose?()
+      atom.packages.deactivatePackages()
+      atom.packages.unloadPackages()
+
+    it 'dispatches the grammar-used command', ->
+      called = false
+      registration = atom.commands.add editor, 'language-coffee-script:grammar-used', (event) ->
+        console.log event
```

Comment:
Yes, console.log shouldn't be there. This is WIP, I'm moving some stuff around and will clean up history.